packages/ui/src/components/QueryBuilder/index.tsx

```diff
@@ -111,7 +111,7 @@ const QueryBuilder: React.FC<IQueryBuilderProps> = ({
         onUpdate(getUpdatedState(queries, activeQuery));
     }, [queries, activeQuery]);
     return (
-        <StackLayout className={`${styles.container} ${className}`} vertical>
+        <StackLayout className={`${styles.container} ${className}`}>
```

Comment:
you cant remove the vertical. Button will be on the right if you do that:

![image](https://user-images.githubusercontent.com/74249528/112310307-cee5d680-8c7a-11eb-9abf-782de3bb1313.png)

Reply:
I'll fix that